src/distributions/bernoulli.rs

```diff
@@ -60,6 +64,18 @@ impl Bernoulli {
         };
         Bernoulli { p_int }
     }
+
+    /// Construct a new `Bernoulli` with the probability of success given as an
+    /// integer `p_int = p * 2^64`.
+    ///
+    /// `p_int = 0` corresponds to `p = 0.0`.
+    /// `p_int = u64::MAX` corresponds to `p = 1.0`.
+    ///
+    /// This is more precise than using `Bernoulli::new`.
+    #[inline]
+    pub fn from_int(p_int: u64) -> Bernoulli {
```

Comment:
I don't think this makes much sense to expose — if users have a probability expressed as a `u64` it's quite easy to implement an appropriate test anyway, and users can deal with any bias as appropriate themselves. Our mapping of 1.0 to `u64::MAX` is essentially a hack, taking advantage of the fact that this number could not be generated otherwise.